packages/runner/src/lib/SecScan.ts

```diff
@@ -0,0 +1,47 @@
+import { Configuration } from '@secbox/core';
+import { Reporter } from '@secbox/reporter';
+import {
+  ScanFactory,
+  ScanSettingsOptions,
+  Severity,
+  TargetOptions
+} from '@secbox/scan';
+
+export class SecScan {
+  private readonly scanFactory: ScanFactory;
+  private readonly reporter: Reporter;
+  private _threshold = Severity.LOW;
+
+  constructor(
+    private readonly configuration: Configuration,
+    private readonly settings: Omit<ScanSettingsOptions, 'target'>
+  ) {
+    this.scanFactory = this.configuration.container.resolve(ScanFactory);
+    this.reporter = this.configuration.container.resolve(Reporter);
+  }
```

Comment:
Why don't pass all necessary dependencies explicitly, especially if we don't use `configuration`?
```suggestion
  constructor(
    private readonly settings: Omit<ScanSettingsOptions, 'target'>,
    private readonly scanFactory: ScanFactory,
    private readonly reporter: Reporter
  ) {
  }
```